scripts/systems/dnd5e.js

```diff
@@ -1,33 +1,43 @@
 import { logger } from "../logger.js";
 import { settings } from "../settings.js";
 
-export function register_helper()
-{
-  logger.info(`Registering DND5E Helpers`);
-  /*
-    Override
-  */
-  game.dnd5e.macros.rollItem = function(itemName){
-    const speaker = ChatMessage.getSpeaker();
-    let actor;
-    if ( speaker.token ) actor = game.actors.tokens[speaker.token];
-    if ( !actor ) actor = game.actors.get(speaker.actor);
+export function register_helper(){
+    logger.info(`Registering DND5E Helpers`);
+    
+    /*
+        Override
+    */
+    const itemMacroUseItem = function(name) {
```

Comment:
Gate this function substitution by `settings.isV10` for safety.

Spit out console warning about functionality being disabled in v9.

Reply:
I am not able to test in v9 currently, but the same magic could likely be done there - restoring v9 functionality by doing the same to `game.dnd5e.macro.rollItem`. Or should it rather just be disabled completely?

Reply:
Let's disable completely. Its been broken for the entire lifespan of DnD 1.5+, iirc. No sense in going back now.